packages/ui/src/components/dialog/dialog-footer.web.tsx

```diff
@@ -0,0 +1,26 @@
+import { Flex, FlexProps } from 'leather-styles/jsx';
+
+interface DialogFooterProps extends FlexProps {
+  children: React.ReactNode;
+}
+
+export function DialogFooter({ children, ...props }: DialogFooterProps) {
+  return (
+    <Flex
+      gap="space.05"
+      p="space.05"
+      bottom={0}
+      width="100vw"
+      maxWidth="100%"
```

Comment:
just `width: 100%` is not the same? 🤔

Reply:
Thanks @alter-eggo ✋ 

You're right, using `width: 100%` works here. I tested it and updated it